docs/Basics-Dimensions.md

```diff
@@ -0,0 +1,63 @@
+---
+id: basics-dimensions
+title: Dimensions
+layout: docs
+category: The Basics
+permalink: docs/basics-dimensions.html
+next: basics-layout
+---
+
+A component's dimensions determine its size on the screen.
+
+#### Fixed Dimensions
+
+The simplest way to set the dimensions of a component is by adding a fixed `width` and `height` to style. All dimensions in React Native are unitless, and represent density-independent pixels.
+
+```ReactNativeWebPlayer
+import React from 'react';
+import { AppRegistry, View } from 'react-native';
+
+class AwesomeProject {
```

Comment:
I was thinking the same thing too. Since this has been accepted and started to ship, I will create a PR to make the `extends` on these. No need to block this now, imho.